### ISSUE TITLE
Location dropdown uses validation logic for VMs

### ISSUE THREAD

**Issue by noelbundick** [Azure/aci-issues#18]
I'm trying to create some containers via the Azure portal. When I select WestUS, I get the following error:

![image](https://user-images.githubusercontent.com/1970162/40949728-47d79ff2-6824-11e8-89eb-1fa54a542966.png)

This puzzles me, because I'm not trying to create a VM. When I run the following command via azure-cli in the cloud shell, it works just fine

```bash
az container create -g vnet2-vnet -n nginx --image nginx -l westus --ip-address public --ports 80
```

It seems like perhaps the wrong control, or the wrong set of options are being passed to the control in the portal?

**Comment by jluk** [comment]
Yep I think this is due to the quota API not recognizing ACI as a resource last time I recall speaking with Jenna - adding @chackdan who is the new ACI owner so he is aware of this issue and repo.
